.github/ISSUE_TEMPLATE/plugin_question.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Plugin related question
+about: I have a question whether I can use certain plugins in flutter-desktop
+title: ""
+labels: ""
+assignees: ""
+---
+
+<!--
+You can find list of available plugins [here](https://github.com/google/flutter-desktop-embedding/tree/master/plugins)
+Note: Most of the flutter plugins are depending on android and ios native SDKs, so most likely you won't be able to use most of flutter plugins as-is
```

Comment:
I think this is misleading.
- First, by definition no existing mobile plugin can be used as-is. E.g., SQLite is available pretty much everywhere, but you still can't use the iOS or Android sqflite plugin implementation on any desktop platform.
- Second, I think it conflates "is this currently available" with "could this be written". We only currently have `path_provider` for macOS, for instance, but someone could write the other platforms at any point, and we in fact have a bug tracking that. That's completely different than questions about Firebase, where the underlying SDK does not itself support desktop.

I don't think there's a good way to answer the general category of questions in a way that's short enough that people will actually read it.